src/pid/status.rs

```diff
@@ -186,6 +187,9 @@ named!(parse_status_state<State>,
           | tag!("Z (zombie)") => { |_| State::Zombie }));
 
 named!(parse_command<String>,   delimited!(tag!("Name:\t"),      parse_line,         line_ending));
+#[cfg(all(target_os = "android", not(target_arch = "aarch64")))]
```

Comment:
So it's a `u16` on non-ARM android targets, and otherwise a `u32`?  Strikes me as strange, but I'm not too familiar with Android or Arm.

Reply:
So I've only thought about Arm Android (u16 on Arm, u32 on Arm64) not sure if this will work on x86 or mips android.

Reply:
OK, if you aren't sure if the arch matters, could you simplify the cfg pairs to  `#[cfg(target_os="android")]` and `#[cfg(not(target_os="android"))]`?  I'd rather keep it simple, given we don't know what the right answer is.

Reply:
Need to leave the aarch64 in otherwise it won't build for aarch64. I'll try fix it for x86 Android.